checkstyle/suppressions.xml

```diff
@@ -44,7 +44,7 @@
     <suppress checks="MethodLength"
               files="(KafkaClusterTestKit).java"/>
     <suppress checks="JavaNCSS"
-              files="RemoteLogManagerTest.java"/>
+              files="(RemoteLogManagerTest|SharePartitionTest).java"/>
```

Comment:
Hi Apoorv, I tried some small refactors like having a function assertBatchEquals / acquireRecords ([commit](https://github.com/apache/kafka/pull/16459/commits/ca1cf68dd76e57b416f9b004bba717c1871ca33f)), but they were too small an improvements to remove the NCSS issue. Reading about this checkstyle, it takes into account the total lines of non-commented source code. So, when we add new test cases, this might come again, hence added the suppression.

Reply:
Is it's not required now, then remove it. In future if it happens again then we can add it back.

Reply:
it is required right now as well, hence keeping it. Thanks.